en/README.md

```diff
@@ -1,15 +1,13 @@
+<div style="float:right; padding:10px; margin-right:20px;"><a href="http://px4.io/"><img src="../assets/site/logo_pro_small.png" title="PX4 Logo" width="180px"/></a></div>

```

Comment:
This adds PX4 logo and link to site to front page. Provides context.  The remainder of the changes aim to make it look just a bit more professional. 

Ultimately I hope to create a more detailed use-case based entry point here, but the rest of the organisation still needs too much work for that.